test/model/model_test.rb

```diff
@@ -153,6 +168,23 @@ class Event
         assert_equal 'text/plain', event.send(:__robject).content_type
         Event.content_type 'application/json'
       end
+
+      test 'persist the object with quorum options' do
+        event = Event.create(name: 'Ruby')
+
+        q = { w: 1, dw: 1 }
+        event.send(:__robject).expects(:store).with(q).once
+        event.save(quorum: q)
+      end
+
+      test 'raises an error with invalid quorum options' do
+        event = Event.create(name: 'Ruby')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.